addons/sales_team/models/crm_team.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+
+
+class CrmTeam(models.Model):
+    _name = "crm.team"
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    _description = "Sales Team"
+    _order = "name"
+
+    @api.model
+    def _get_default_team_id(self, user_id=None):
+
+        if user_id is None:
+            user_id = self.env.uid
+
+        team_id = self.env.context.get('default_team_id')
+        if not team_id:
+            team = self.search(['|', ('user_id', '=', user_id), ('member_ids', 'in', [user_id])], limit=1)
+            team_id = team.id if team else self.env.ref('sales_team.team_sales_department').id
+        return team_id
+
+    name = fields.Char('Sales Team', size=64, required=True, translate=True)
```

Comment:
Side question: is it really necessary to limit the size of the sales team name ?